src/aio/aio_theme_switch.py

```diff
@@ -18,25 +11,30 @@ class ids:
             "subcomponent": "switch",
             "aio_id": aio_id,
         }
-        store = lambda aio_id: {
+        leftIcon = lambda aio_id: {
             "component": "ThemeSwitchAIO",
-            "subcomponent": "store",
+            "subcomponent": "leftIcon",
             "aio_id": aio_id,
         }
-        dummy_div = lambda aio_id: {
+        rightIcon = lambda aio_id: {
             "component": "ThemeSwitchAIO",
-            "subcomponent": "dummy_div",
+            "subcomponent": "rightIcon",
+            "aio_id": aio_id,
+        }
+        store = lambda aio_id: {
+            "component": "ThemeSwitchAIO",
+            "subcomponent": "store",
             "aio_id": aio_id,
         }
 
     ids = ids
 
     def __init__(
-        self,
-        themes=None,
-        icons=None,
-        switch_props={},
-        aio_id=None,
+            self,
+            aio_id: str = str(uuid.uuid4()),
+            themes: tuple[str, str] | str = (dbc.themes.CYBORG, dbc.themes.BOOTSTRAP),
```

Comment:
```suggestion
            themes: tuple[str, str] | list[str] = (dbc.themes.CYBORG, dbc.themes.BOOTSTRAP),
```

It's a nice idea to make it so this component will use the Bootstrap color modes if there is only one theme.  However I'd like to keep this component where the use-case is to switch entire stylesheets, which is a little more complex.  When using color modes, all that's necessary is a one line clientside callback